src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java

```diff
@@ -12,10 +14,23 @@
     private static final int SCALE_HEIGHT = Space.DEFAULT_HEIGHT;
     private static final int SIZE_WINDOW_WIDTH = SCALE_WIDTH * SIZE_IMAGE;
     private static final int SIZE_WINDOW_HEIGHT = SCALE_HEIGHT * SIZE_IMAGE;
+    private static final int NUMBER_HUNDRED = 100;
 
     private static JLabel[][] labelArray;
     private Spaceship spaceship;
 
+    private Timer timer;
+    private int tic;
+    public static final int DELAY_OF_CYCLE_IN_MILISECONDS = 10;
+
+    public static final int VELOCITY_GROUP_ALIEN = 50;
+
```

Comment:
maybe remove the spaces between variables

Reply:
okay 👍🏻

Reply:
Already solved 😁